telegram/_bot.py

```diff
@@ -3716,7 +3716,7 @@ async def set_webhook(
 
         If you'd like to make sure that the Webhook request comes from Telegram, Telegram
         recommends using a secret path in the URL, e.g. https://www.example.com/<token>. Since
-        nobody else knows your bot's token, you can be pretty sure it's us.
+        nobody else knows your bot's token, you can be pretty sure it's them.
```

Comment:
```suggestion
        nobody else knows your bot's token, you can be pretty sure it's Telegram.
```
I would change it to `Telegram` since it makes more clear who it is, rather than `us` (PTB or Telegram?) and `them` (Telegram or the attackers?)

Reply:
I'm fine with both.

Reply:
Please @ me when you have agreed on something, I'm fine with both, but slightly prefer `them` because it's been mentioned several times in the sentence before that we're talking about Telegram.